### ISSUE TITLE
Bad handshake with Stargate

### ISSUE THREAD

**Issue by rmontagnin** [forbole/juno#6]
## Description
As reported inside forbole/bdjuno#55, Juno seems to not work when trying to connect to Stargate. 

## Branch
```
cosmos-v0.39.x
```

## Configuration
```toml
rpc_node = "http://rpc.stargate.bigdipper.live:80"
client_node = "http://api.stargate.bigdipper.live:443"
```

## Log
```
{"level":"debug","time":"2020-10-08T09:28:50+02:00","message":"Reading config file"}
ERROR: websocket: bad handshake: failed to start RPC client
```

**Comment by rmontagnin** [comment]
After taking a look, it seems to me that this is due to the fact that Stargate uses gRPC instead of the classic RPC.

I've tried creating a new branch called `cosmos-stargate-3`. On it, I've updated Cosmos to use the branch `zaki/stargate-3`. 

Unfortunately, this branch seems not to compile properly. The returned error is the following: 

```
$ make build
building juno binary...
../../Go/pkg/mod/github.com/cosmos/cosmos-sdk@v0.34.4-0.20200921130040-27db2cf89772/client/grpc/reflection/reflection.pb.go:9:2: cannot find module providing package github.com/gogo/protobuf/grpc: import lookup disabled by -mod=readonly
make: *** [Makefile:23: build] Error 1
```

I think we need to mark this as stale and leave it until a stable version of Cosmos v0.40 is out and we can use it properly.

**Comment by kwunyeung** [comment]
Just noticed the configurations listed here is not correct. Both are running with TLS and the URLs should be

```
rpc_node = "https://rpc.stargate.bigdipper.live:443"
client_node = "https://api.stargate.bigdipper.live:443"
```

**Comment by rmontagnin** [comment]
Closed with branch [`cosmos-v0.40.x`](https://github.com/desmos-labs/juno/tree/cosmos-v0.40-x)
